contracts/cw3-fixed-multisig/src/error.rs

```diff
@@ -1,11 +1,15 @@
 use cosmwasm_std::StdError;
+use cw3_flex_multisig::error::ContractError as Cw3FlexMultisigError;
 use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq)]
 pub enum ContractError {
     #[error("{0}")]
     Std(#[from] StdError),
 
+    #[error("{0}")]
+    FlexMultisig(#[from] Cw3FlexMultisigError),
```

Comment:
Extracting whole `Threshold` implementation in https://github.com/CosmWasm/cw-plus/pull/590 will make this error clearer.